Remotion/Web/Development.WebTesting/Configuration/WebTestConfigurationSection.cs

```diff
@@ -190,6 +197,11 @@ public string LogsDirectory
       get { return (string) this [_logsDirectoryProperty]; }
     }
 
+    public ChromeDisableSecurityWarningsBehavior ChromeDisableSecurityWarningsBehavior

```

Comment:
the discussion on whether to create a nested element isn't resolved yet.

Reply:
This is the only chrome-specific setting at the moment, and I could not think of any other coming in the near future. I do not see the need of adding a nested element atm.

Reply:
Okay, I want a sub element "chrome". The problem is, should we ever have the need for other / additional settings, placing the settings on the root element now would result in a lock-in or breaking change.